private/buf/bufcli/env.go

```diff
@@ -84,3 +86,18 @@ func WarnBetaCommand(_ context.Context, container appext.Container) {
 		container.Logger().Warn("This command is in beta. It is unstable and likely to change. To suppress this warning, set " + betaSuppressWarningsEnvKey + "=1")
 	}
 }
+
+// CheckBetaPluginEnabled checks whether beta lint/breaking plugin is enabled through an env var.
+// This returns nil if it is enabled and an error otherwise.
+//
+// TODO: remove this as part of publicly releasing lint/breaking plugins
+func CheckBetaPluginEnabled(container appext.Container) error {
```

Comment:
IsPluginsEnabled, per the spec

Reply:
This is also supposed to be a bool. You are returning an error saying that there are plugins set, but this function does not actually check if plugins are set or not.